### DISCUSSION TITLE
Import visibility from Maya

### DISCUSSION THREAD

**Discussion by LWestling** [assimp/assimp#4370] [Q&A] [answered]
In Maya you can animate and keyframe the visibility, so you can create an animation where for example part of the mesh is invisible for a few frames, where can you get this data in assimp?

**Comment by kimkulling** [comment, marked as answer]
At this moment this feature is not supported as far as I know. Maybe you can check the metadata for that. If this is not available there maybe you can try to open a feature-request?
